app/routes/campaigns.js

```diff
@@ -2,13 +2,33 @@
 
 const express = require('express');
 const router = express.Router(); // eslint-disable-line new-cap
-
+const Promise = require('bluebird');
+const contentful = require('../../lib/contentful');
 const helpers = require('../../lib/helpers');
 const mobilecommons = rootRequire('lib/mobilecommons');
 const logger = app.locals.logger;
 const stathat = app.locals.stathat;
 
 /**
+ * Queries Contentful to add keywords property to given campaign object.
+ */
+function formatCampaignDoc(campaignDoc) {
+  const campaign = campaignDoc.formatApiResponse();
+
+  return new Promise((resolve, reject) => {
+    logger.debug(`fetchKeywordsForCampaign:${JSON.stringify(campaign)}`);
+
+    return contentful.fetchKeywordsForCampaignId(campaign.id)
```

Comment:
This branch has a slight modification to the `keywords` property in the Gambit `campaigns` API -- it's now returned as an array of objects instead of an array of strings. This is for future-proofing the response -- eventually we want to support overriding CampaignBot messaging per keyword: e.g. A user who texts in a keyword PUPPET and get back a greeting that says "Hey, it's me Puppet Sloth!" instead of the typical greeting. The override would be defined as a new field on the `keyword` content type.